cmd/api-errors.go

```diff
@@ -561,6 +562,11 @@ var errorCodeResponse = map[APIErrorCode]APIError{
 		Description:    "Range specified is not valid for source object",
 		HTTPStatusCode: http.StatusBadRequest,
 	},
+	ErrMetadataTooLarge: {
+		Code:           "InvalidArgument",
+		Description:    "Your metadata headers exceed the maximum allowed metadata size.",
```

Comment:
If we intend to have the string representation of `errMetadataTooLarge` same as `ErrMetadataTooLarge`'s description, then we should do the following,
```go
ErrMetadataTooLarge: {
	Code:           "InvalidArgument",
	Description:    errMetadataTooLarge.String(),
	HTTPStatusCode: http.StatusBadRequest,
},
```